corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -139,6 +140,27 @@ hqDefine("data_dictionary/js/data_dictionary", [
             return self.toBeDeprecated() && !deprecated && self.properties().length > 0;
         });
 
+        self.newCaseProperty = function () {
+            if (_.isString(self.newPropertyName()) && self.newPropertyName().trim()) {
```

Comment:
In what scenario will the conditional `_.isString()` fail? Since this is coming from an input field, I expect this to always be a string.

Reply:
Good call. 9b3efb663d9